docs/sources/getting-started/glossary.md

```diff
@@ -40,12 +40,30 @@ This topic lists words and abbreviations that are commonly used in the Grafana d
       Explore allows a user to focus on a building a query. User can refine the query to return the expected metrics before building a dashboard.
     </td>
   </tr>
+  <tr>
+    <td style="vertical-align: top">Exporter</td>
+    <td>
+      An exporter implements the translation of data that comes out of a data source into a format that Prometheus can digest.
+    </td>
+  </tr>
+  <tr>
+    <td style="vertical-align: top">Integration (Grafana Cloud)</td>
+    <td>
+      Each Integration in Grafana Cloud uses the cloud agent to connect your data source to Grafana Cloud for visualizing. Note: Prometheus uses the word “integrations” to refer to software that exposes Prometheus metrics without needing an exporter, which is a different use of the same word we use here.
```

Comment:
I suggest asking @jhunthrop for his input, because there are several definitions of "integrations" in the Grafana context.

Reply:
I think this entry is pretty clear that it is in the Grafana Cloud context, with some disambiguation provided for the Prometheus context. If/when we learn more, splitting this entry into two and adding a third for how Grafana uses the term differently outside of cloud might be our best option.

Reply:
Fair enough.